src/Drawer/Drawer.js

```diff
@@ -72,6 +72,7 @@ export default class Drawer extends Component {
     docked: PropTypes.bool,
     enterTransitionDuration: PropTypes.number,
     leaveTransitionDuration: PropTypes.number,
+    onRequestClose: PropTypes.func,
```

Comment:
I think that you made a good call. Wrapping component should be exposing critical properties.
I know that it's mean duplication and violation of the DRY principal, but we haven't found better solution.